test/server/cards/01_SOR/upgrades/LukesLightsaber.spec.js

```diff
@@ -0,0 +1,47 @@
+describe('Luke\'s Lightsaber', function() {
+    integration(function() {
+        describe('Luke\'s Lightsaber\'s ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['lukes-lightsaber'],
+                        groundArena: [{ card: 'luke-skywalker#jedi-knight', damage: 5, upgrades: ['shield'] }, { card: 'battlefield-marine', damage: 2 }, 'reinforcement-walker'],
```

Comment:
Just curious, any specific reason to start Luke with a shield on him?

Reply:
An off-chance having an existing upgrade would randomly mess things up.